### DISCUSSION TITLE
Add documentation for selecting code

### DISCUSSION THREAD

**Discussion by nikero41** [CopilotC-Nvim/CopilotChat.nvim#220] [Ideas]
It would be nice to have in the documentation how you can have as a selection a highlighted code section or the whole buffer by default.

I can't understand when it selects the whole buffer or just one line.

I think I can should use the selection field but it would be nice to be in the readme as well.

**Comment by deathbeam** [comment]
It is in README in the default configuration section here:

![image](https://github.com/CopilotC-Nvim/CopilotChat.nvim/assets/5115805/a815e8eb-c4c1-41b4-a679-9c5c90094bb1)

Also in MIGRATION guide:
https://github.com/CopilotC-Nvim/CopilotChat.nvim/blob/canary/MIGRATION.md

And also in API section:

![image](https://github.com/CopilotC-Nvim/CopilotChat.nvim/assets/5115805/792b16dc-fbd5-4af2-85ab-7227c9986868)

Not sure if having it mentioned 4th time is super necessary, is it rly that confusing?

**Comment by jellydn** [comment]
Hi @snikoletopoulos Is that clear for you now or any comments/questions on how it works?

**Comment by nikero41** [comment]
Sorry for my late response.
It is clear yes.
When I try to open the chat with `'<,'>CopilotChatOpen` in a selected range it errors `E481: No range allowed`
Is this expected?

I've noticed if i remove the range it still selects the text

It would be nice if it supported ranges as well
